ui/css/perf.css

```diff
@@ -290,3 +290,8 @@ to { opacity: 0; }
   height: 1em;
   margin-bottom: 5px;
 }
+
+.run-detail {
+    border-bottom: 1px dotted #777;
```

Comment:
In case this helps, I didn't seem to have problems hovering without this underline/border on geomean and confidence. If we wanted we could alternately add luminance (eg. `color: #999;`) if the goal is to provide differentiation between it and the time value?

![geomeans](https://cloud.githubusercontent.com/assets/3660661/8573731/301b96b0-2561-11e5-8e44-e9e9825c6b08.jpg)